Text/CustomNER/README.md

```diff
@@ -1,35 +1,81 @@
+| TA_KEY          | The access key to be able to access the endpoint.               | Can be found under `Resource Management -> Keys and Endpoint` in the Language resource page in Azure Portal. |
+| PROJECT_NAME    | The name of the created project in the previous step.           |                                                                                                              |
+| DEPLOYMENT_NAME | The name of the deployment of the project in the previous step. |                                                                                                              |
+
+After creating an Azure App resource, the Custom NER powerskill can be deployed to the Azure App. This is commonly done in two ways. One approach is to create a local project in Visual Studio Code and deploy it using the Azure Functions extension (more details can be found at the previously linked article). Another approach is to do a ___Zip Deploy__ i.e. upload a zipped file containing the function code and configuration (with a similar structure to [this](https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-python#folder-structure)). An app that is Zip-Deployed can be setup to either [run from a package file](https://docs.microsoft.com/en-us/azure/azure-functions/run-functions-from-deployment-package) or [do a remote build](https://docs.microsoft.com/en-us/azure/azure-functions/run-functions-from-deployment-package). Running from a package assumes that the project is ready to be run and skips doing any build steps (e.g. `npm install`, or in this case, `pip install`). Since this Python project needs to pull dependencies with pip, a remote build is the more appropriate choice. A simple command to zip the necessary files would be
+
+```bash
+ zip -r customner-powerskill.zip custom_ner host.json requirements.txt
+```
+
+For more information on deployment methods, see [Deployment technologies in Azure Functions | Microsoft Docs](https://docs.microsoft.com/en-us/azure/azure-functions/functions-deployment-technologies).
+
+At this point, you should have a working Azure Function. Depending on how you deployed the function, you may need to add a `x-functions-key` header to each request to the function endpoint. The value for the header can be found in `Functions -> App keys` in the Function App resource page in Azure Portal.
+
+The function adhers to the input/output format specified by Azure Cognitive Search for custom skills. More information about custom skills and format, see [Custom skill interface - Azure Cognitive Search | Microsoft Docs](https://docs.microsoft.com/en-us/azure/search/cognitive-search-custom-skill-interface). Sample inputs and outputs for this powerskill are shown below. Notice that specifying the language is optional, and defaults to English.
```

Comment:
typo: "adhers" -> "adheres"

Reply:
Fixed in [a417a1b](https://github.com/Azure-Samples/azure-search-power-skills/pull/97/commits/a417a1b3c3c6dae7c6056b620329136fc4d4b12c)